projects/batfish/src/org/batfish/grammar/cisco/Cisco_ignored.g4

```diff
@@ -410,6 +411,7 @@ null_block_substanza
          | ERSPAN_ID
          | ESCAPE_CHARACTER
          | EXCEED_ACTION
+         | EXEC
```

Comment:
I really need to see a unit test here. This might conflict with other changes I've made. In fact I'm pretty sure I removed this keyword from here recently.